lua/wire/wireshared.lua

```diff
@@ -999,3 +999,71 @@ function nicenumber.nicetime( n )
 		return "0s"
 	end
 end
+
+------------------------------------------------------
+-- Loads a font into the cache if it doesn't exist
+-- Returns true if font exists, else returns false
+------------------------------------------------------
+local _fontcache = {}
+local _defaultFontSize = 18
+
+function WireLib.LoadFont( name, size )
+	if not name then return nil end
+	
+	local fontSize = size or _defaultFontSize
+	local fontKey = name .. "_" .. fontSize
+	
+	-- Shortcircuit if font has already loaded
+	if _fontcache[fontKey] ~= nil then
+		return fontKey
+	end
+	
+	-- If font isn't in cache, create it
+	local fontTable =
+	{
+		font = name,
+		size = fontSize,
+		weight = 800,
```

Comment:
I added them because they were already being used by both the EGP and the GPU font loading code. If they don't break anything then my all means remove them, but I didn't want to go removing them in case they were important.